apps/site/src/content/blog/state-of-the-budgie-2023.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: "State of the Budgie: 2023 In Review and Goals for 2024"
+author: JoshuaStrobl
+publishDate: "2024-01-26T19:00:00.000+02:00"
+featuredImage: /images/blog/2024/01/StateOfTheBudgieY23.png
+excerpt: >-
+  2023 was an amazing year for the Budgie Desktop and our organization. Time to dive into our State of the Budgie to recap our efforts in 2023 and the goals for 2024.
+tags:
+  - state-of-the-budgie
+  - news
+---
+
+## 2023
+
+2023's theme was centered around our "User-centric" organization value. We focused on this value in a couple important ways:
+
+1. Delivering major new features and architectural improvements across Budgie 10.7 and 10.8 releases.
+2. Improving stability and UX consistency with bug fixes and new components such as Magpie (our fork of Mutter 43.x).
+
+Across 2023, we had a total of 15 releases spanning our microcosm of projects in our organization and the addition of 3 new projects (budgie-backgrounds, budgie-session, and magpie).
+
+Additionally, the team significantly shifted its development focus in mid 2023 towards moving Budgie 10 to being Wayland-only. While this did mean that some efforts on Budgie 11 were deferred to this year, it also accelerated other areas with a focus on tackling one of the biggest components (Wayland compositor) that could be used for both Budgie 10 and 11.
+
+### Budgie 10
+
+Budgie Desktop saw a total of 6 releases of Budgie 10.x throughout 2023. We kicked off the year with Budgie 10.7 in January, with bug fixes and QoL improvements in February and April. In August, we released Budgie 10.8 and followed up with two bug fix releases in 10.8 series in October. The rest of the year was focused on porting various components from using libwnck directly to libxfce4windowing for Wayland support and of course some much deserved rest for the holidays!
+
+While I will refrain from exhaustively writing about all the amazing work that has gone into those releases(or else this will turn into the Odyssey by Homer), we will highlight some areas, with snippets from the respective blog posts.
```

Comment:
missing space between `releases` and parenthesis

Reply:
Resolved with 055e4734eb9e38b2848c95cbf82af23ceb42c99f